sockeye/inference.py

```diff
@@ -820,12 +820,12 @@ def __init__(self,
                  ensemble_mode: str,
                  length_penalty: LengthPenalty,
                  batch_size: int,
-                 beam_size: int,
                  beam_prune: float,
                  beam_search_stop: str,
                  models: List[SockeyeModel],
                  source_vocabs: List[vocab.Vocab],
                  target_vocab: vocab.Vocab,
+                 beam_size: int = 5,
```

Comment:
Can we remove the default value here again and explicitly create it on CheckpointDecoder with beam_size=5? I like the user to be forced to think about beam size when constructing a Translator.

Reply:
Why? I think it would be good to simply have a reasonable default with the option to change it. More often than not we don't want to change the beam size, no?

Reply:
If we force the user to think about less important parameters such as beam_prune and beam_search_stop, I think we can ask her/him for a reasonable beam_size :)

Reply:
maybe we should add more defaults also for those parameters. :)

Reply:
Ok, but then we need to keep them in sync with the default values in arguments.py, which I find tedious :)
It's fine for now, but we should do a pass over the code for such small things at some point.